playlistmanager.lua

```diff
@@ -461,7 +464,19 @@ end
 function draw_playlist()
   refresh_globals()
   local ass = assdraw.ass_new()
-  ass:new_event()
+
+  local w, h = mp.get_osd_size()
+
+  if settings.curtain_opacity ~= nil and settings.curtain_opacity < 1.0 then
+  -- From https://github.com/christoph-heinrich/mpv-quality-menu/blob/501794bfbef468ee6a61e54fc8821fe5cd72c4ed/quality-menu.lua#L699-L707
```

Comment:
This is copyleft licensed code. I don't think we can copy it directly.

Reply:
Permission was given to add it to another script, so I assumed it would be fine to add to this one. See: https://github.com/christoph-heinrich/mpv-quality-menu/commit/501794bfbef468ee6a61e54fc8821fe5cd72c4ed